src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,28 @@
-export const TodoInfo = () => {};
+import usersFromServer from '../../api/users';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = getUser(todo.userId);
+
+  return (
+    <article data-id="1" className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
```

Comment:
FYI: it's better to use `classnames` library in case when you have conditional className